Update API Integration tests due to test dependency update

|Wazuh version|Component|
|---|---|
| 4.7.0 and upwards | API endpoints Integration Tests |

## Description
<!--
Whenever possible, issues should be created for bug reporting and feature requests.
For questions related to the user experience, please refer:
- Wazuh mailing list: https://groups.google.com/forum/#!forum/wazuh
- Join Wazuh on Slack: https://wazuh.com/community/join-us-on-slack
-->
The current dependencies required to run the API endpoints integration tests include the `tavern` module. The version used until Wazuh version `4.6.0` was `1.2.2`, but since `4.7.0`, the [requirements-dev.txt file](https://github.com/wazuh/wazuh/blob/4.7.0/framework/requirements-dev.txt) introduced `tavern` `1.23.5` and `pytest` `7.3.1`. Due to these updates, the tests `test_rbac_black_cluster_endpoints.tavern.yaml` and `test_rbac_white_cluster_endpoints.tavern.yaml` raise the following type of  error and fail in some of its test cases:
<details><summary>AIT error</summary>

    ____________________________________________ /home/fdalmau/git/wazuh/api/test/integration/test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/status ____________________________________________
    Format variables:
      worker1 = '???'
      protocol:s = '???'
      host:s = '???'
      port:d = '???'
      test_login_token = '???'
      worker2 = '???'
      protocol:s = '???'
      host:s = '???'
      port:d = '???'
      test_login_token = '???'
    
    Source test stages:
      - name: Request {worker1} (Allow)
        request:
          verify: False
          url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/status"
          method: GET
          headers:
            Authorization: "Bearer {test_login_token}"
        response:
          status_code: 200
    
      - name: Request {worker2} (Deny)
        request:
          verify: False
          url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/status"
          method: GET
          headers:
            Authorization: "Bearer {test_login_token}"
        response:
          <<: *permission_denied
    
    
    Stage not found
    
    Errors:
    E   tavern.util.exceptions.MissingFormatError: worker1
    ------------------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------------------
    ERROR    tavern.util.dict_util:dict_util.py:36 Failed to resolve string 'Request {worker1} (Allow)' with variables '{'protocol': 'https', 'host': 'localhost', 'port': 55000, 'user': 'testing', 'user_auth_context': 'wazuh-wui', 'pass': 'wazuh', 'basic_auth': 'dGVzdGluZzp3YXp1aA==', 'basic_auth_context': 'd2F6dWgtd3VpOndhenVoLXd1aQ==', 'login_endpoint': 'security/user/authenticate', 'reconnect_delay': 30, 'restart_delay_cluster': 150, 'upgrade_delay': 60, 'global_db_delay': 30, 'cluster_sync_delay': 20, 'custom_wpk_path': '/var/ossec/test.wpk', 'new_rules': '<!-- Local rules -->\n <group name="local,">\n    <!--   NEW RULE    -->\n    <rule id="111111" level="5">\n      <if_sid>5716</if_sid>\n      <srcip>1.1.1.1</srcip>\n      <description>sshd: authentication failed from IP 1.1.1.1.</description>\n      <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n    </rule>\n  </group>\n', 'new_rules_with_syntax_error': '<!-- Local rules -->\n <group name="local,">\n <!--   NEW RULE    -->\n <rule id="111111" level="5">\n <if_sid>5716\n <srcip>1.1.1.1</srcip>\n  <description>sshd: authentication failed from IP 1.1.1.1.</description>\n <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n </rule>\n </group>\n', 'new_decoder': '<!-- NEW Local Decoders -->\n <decoder name="local_decoder_example">\n <program_name>NEW DECODER</program_name>\n </decoder>\n', 'new_decoder_with_syntax_error': '<!-- NEW Local Decoders -->\n <decoder name="local_decoder_example" NEW DECODER</program_name>\n </decoder>\n', 'new_cdb_list': 'test-wazuh-w:write\ntest-wazuh-r:read\ntest-wazuh-a:attribute\ntest-wazuh-x:execute\ntest-wazuh-c:command\n', 'new_cdb_list_with_syntax_error': ':write\ntest-wazuh-r:read\ntest-wazuh-a:attribute\ntest-wazuh-x:execute\ntest-wazuh-c:command\n', 'corrupted_rules_file': '<!-- Local rules -->\n <group name="local,">\n <!--   NEW RULE    -->\n  <rule id="111111" level="XXX">\n      <if_sid>5716</if_sid>\n      <srcip>1.1.1.1</srcip>\n      <description>sshd: authentication failed from IP 1.1.1.1.</description>\n      <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n    </rule>\n  </group>\n', 'invalid_file': "import os #:l\nos.system('do something bad') #:l", 'file_xml': '<agent_config> <labels> <label key="aws.instance-id">i-052a1838c</label> <label key="aws.sec-group">sg-1103</label> <label key="network.ip">172.17.0.0</label> <label key="network.mac">02:42:ac:11:00:02</label> <label key="installation" hidden="yes">January 1st, 2017</label> </labels> </agent_config>', 'invalid_file_xml': 'agent_config> <labels> <label key="aws.instance-id">i-052a1838c</label> <label key="aws.sec-group">sg-1103</label> <label key="network.ip">172.17.0.0</label> <label key="network.mac">02:42:ac:11:00:02</label> <label key="installation" hidden="yes">January 1st, 2017</label> </labels> </agent_config>', 'wrong_file_xml': '<agent_confi>\n<wodle name="cis-cat"> <disabled>no</disabled> <timeout>3600</timeout> <interval>90d</interval> <scan-on-start>yes</scan-on-start>\n<java_path>/opt/java/bin</java_path> <ciscat_path>/opt/scanner</ciscat_path>\n<content type="xccdf" path="/opt/xccdf.xml"> <profile>level2</profile> </content> </wodle>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-java-check</alias> <command>fi</command> <frequency>86400</frequency> </localfile>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-scanner-check</alias> <command>fi</command> <frequency>86400</frequency> </localfile>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-ossec-log</alias> <command>grep \'wazuh-modulesd:ciscat:\' /opt/ossec/logs/ossec.log | tail -20</command> <frequency>86400</frequency> </localfile>\n</agent_confi>\n<agent_config>\n<!-- Realtime scan -->\n<syscheck>\n<!-- Full directories -->\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/logging</directories>\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/c2online</directories>\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/c2online</directories> <directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/addons</directories> <directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/commons</directories>\n\n<!-- Specific files --> <directories check_all="yes" report_changes="no" realtime="yes" restrict=".war$">/opt/deploy/c2online*.war</directories>\n<!-- Ignore folders --> <ignore type="sregex">.log$</ignore>\n</syscheck>\n</agent_config>', 'query_list_xml': "<agent_config> <localfile> <location>System</location> <log_format>eventchannel</log_format> <query> \\<QueryList> \\<Query Id='0' Path='System'> \\<Select Path='System'>*[System[(Level&lt;=3)]]\\</Select> \\</Query> \\</QueryList> </query> </localfile> </agent_config>", 'valid_ossec_conf': '<ossec_config> <global> <jsonout_output>yes</jsonout_output> <alerts_log>yes</alerts_log> <logall>no</logall> <logall_json>no</logall_json> <email_notification>no</email_notification> <agents_disconnection_time>20s</agents_disconnection_time> <agents_disconnection_alert_time>100s</agents_disconnection_alert_time> </global>\n<logging> <log_format>plain</log_format> </logging>\n<alerts> <log_alert_level>300</log_alert_level> <email_alert_level>12</email_alert_level> </alerts>\n<remote> <connection>secure</connection> <port>1514</port> <protocol>tcp</protocol> </remote>\n<syscheck> <disabled>yes</disabled> </syscheck> </ossec_config>', 'invalid_ossec_conf': '<ossec_config> <global> <jsonout_output>yes</jsonout_output> <alerts_log>yes</alerts_log> <logall>no</logall> <logall_json>no</logall_json> </global>\n<alerts> <log_alert_level>3</log_alert_level> <email_alert_level>12</email_alert_level> </alerts>\n<remote> </ossec_config>', 'api_test': None, 'request': <FixtureRequest for <YamlItem GET /cluster/{node_id}/status>>, 'test_login_token': 'eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjk3MTE0OTc1LCJleHAiOjE2OTcxMTU4NzUsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbOTldLCJyYmFjX21vZGUiOiJibGFjayJ9.AcXud4scJbPHzlWaRzOW7fmZmAVma_U9u0KfQ56ksBhbMw6YYBMs9Wxe4A66OML9tm2e84PsomKJkNcvTnEkEf3NASIK4tHtf7gXQpYvy1HwTdPnMnlUmYfv_P-WyDcxzZBdaUh0KlyqKNr0Wh0cAgAHRBsHwDIZEoosvJfORp1GPPy4', 'tavern': {'env_vars': {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/wazuhFW:@/tmp/.ICE-unix/2294,unix/wazuhFW:/tmp/.ICE-unix/2294', 'QT_ACCESSIBILITY': '1', 'COLORTERM': 'truecolor', 'XDG_CONFIG_DIRS': '/etc/xdg/xdg-ubuntu:/etc/xdg', 'XDG_MENU_PREFIX': 'gnome-', 'GNOME_DESKTOP_SESSION_ID': 'this-is-deprecated', 'TERMINATOR_DBUS_PATH': '/net/tenshu/Terminator2', 'MANDATORY_PATH': '/usr/share/gconf/ubuntu.mandatory.path', 'LC_ADDRESS': 'es_AR.UTF-8', 'GNOME_SHELL_SESSION_MODE': 'ubuntu', 'LC_NAME': 'es_AR.UTF-8', 'SSH_AUTH_SOCK': '/run/user/1000/keyring/ssh', 'TERMINATOR_UUID': 'urn:uuid:a5826204-f1e0-4a1b-9498-6bfce72d494b', 'XMODIFIERS': '@im=ibus', 'DESKTOP_SESSION': 'ubuntu', 'LC_MONETARY': 'es_AR.UTF-8', 'SSH_AGENT_PID': '2256', 'GPG_TTY': '/dev/pts/0', 'GTK_MODULES': 'gail:atk-bridge', 'PWD': '/home/fdalmau/git/wazuh/api/test/integration', 'LOGNAME': 'fdalmau', 'XDG_SESSION_DESKTOP': 'ubuntu', 'XDG_SESSION_TYPE': 'x11', 'GPG_AGENT_INFO': '/run/user/1000/gnupg/S.gpg-agent:0:1', 'XAUTHORITY': '/run/user/1000/gdm/Xauthority', 'GJS_DEBUG_TOPICS': 'JS ERROR;JS LOG', 'WINDOWPATH': '2', 'HOME': '/home/fdalmau', 'USERNAME': 'fdalmau', 'IM_CONFIG_PHASE': '1', 'LC_PAPER': 'es_AR.UTF-8', 'LANG': 'en_US.UTF-8', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'XDG_CURRENT_DESKTOP': 'ubuntu:GNOME', 'VIRTUAL_ENV': '/home/fdalmau/venv/integrationtest-env', 'VTE_VERSION': '6003', 'INVOCATION_ID': 'cbc24b4745a44f0a975da72b5b066c90', 'TERMINATOR_DBUS_NAME': 'net.tenshu.Terminator21a9d5db22c73a993ff0b42f64b396873', 'MANAGERPID': '2073', 'GJS_DEBUG_OUTPUT': 'stderr', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'XDG_SESSION_CLASS': 'user', 'PYTHONPATH': ':/home/fdalmau/git/wazuh/api:/home/fdalmau/git/wazuh/framework', 'TERM': 'xterm-256color', 'LC_IDENTIFICATION': 'es_AR.UTF-8', 'DEFAULTS_PATH': '/usr/share/gconf/ubuntu.default.path', 'LESSOPEN': '| /usr/bin/lesspipe %s', 'USER': 'fdalmau', 'DISPLAY': ':0', 'SHLVL': '1', 'LC_TELEPHONE': 'es_AR.UTF-8', 'QT_IM_MODULE': 'ibus', 'LC_MEASUREMENT': 'es_AR.UTF-8', 'XDG_RUNTIME_DIR': '/run/user/1000', 'PS1': '(integrationtest-env) \\[\\e]0;\\u@\\h: \\w\\a\\]${debian_chroot:+($debian_chroot)}\\[\\033[01;32m\\]\\u@\\h\\[\\033[00m\\]:\\[\\033[01;34m\\]\\w\\[\\033[01;33m\\]$(parse_git_branch)\\[\\033[00m\\]\\$ ', 'LC_TIME': 'es_AR.UTF-8', 'JOURNAL_STREAM': '8:53407', 'XDG_DATA_DIRS': '/usr/share/ubuntu:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PATH': '/home/fdalmau/venv/integrationtest-env/bin:/home/fdalmau/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'GDMSESSION': 'ubuntu', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1000/bus', 'GIO_LAUNCHED_DESKTOP_FILE_PID': '5891', 'GIO_LAUNCHED_DESKTOP_FILE': '/usr/share/applications/terminator.desktop', 'LC_NUMERIC': 'es_AR.UTF-8', 'OLDPWD': '/home/fdalmau/git/wazuh', '_': '/home/fdalmau/venv/integrationtest-env/bin/pytest', 'PYTEST_CURRENT_TEST': 'test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/status (call)'}}}'
    ERROR    tavern.util.dict_util:dict_util.py:39 Key(s) not found in format: worker1


</details>

We should review these tests and others that may require modifications to avoid these types of errors related to the test dependencies update.


## Checks
<!-- Do not modify, this will be ticked during development -->
The following elements have been updated or reviewed (should also be checked if no modification is required):
- [x] Tests (unit tests, API integration tests).
- [x] Changelog.
- [x] Documentation.
- [x] Integration test mapping (using `api/test/integration/mapping/_test_mapping.py`).

# Issue Update

The problem found in the description of the issue is related to how the `tavern` framework formats the curly brackets. The found error is also present in version `1.2.2` but is only found when a test fails. Here is an example when forcing an error:

<details><summary>test_rbac_black_cluster_endpoints.tavern.yaml</summary>

```console
# tavern `1.2.2`

pytest -vvv test_rbac_black_cluster_endpoints.tavern.yaml
====================================================================== test session starts ======================================================================
platform linux -- Python 3.9.18, pytest-5.4.3, py-1.11.0, pluggy-0.13.1 -- /home/fdalmau/venv/integrationtest-env-3.9/bin/python3.9
cachedir: .pytest_cache
metadata: {'Python': '3.9.18', 'Platform': 'Linux-5.15.0-86-generic-x86_64-with-glibc2.31', 'Packages': {'pytest': '5.4.3', 'py': '1.11.0', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'tavern': '1.2.2', 'metadata': '2.0.2'}}
rootdir: /home/fdalmau/git/wazuh/api/test/integration, inifile: pytest.ini
plugins: html-3.1.1, tavern-1.2.2, metadata-2.0.2
collected 19 items                                                                                                                                              

test_rbac_black_cluster_endpoints.tavern.yaml::/cluster/local/config PASSED                                                                               [  5%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/healthcheck PASSED                                                                            [ 10%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/ruleset/synchronization PASSED                                                                [ 15%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/local/info PASSED                                                                             [ 21%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/nodes PASSED                                                                                  [ 26%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/nodes/{node_id} PASSED                                                                        [ 31%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/status PASSED                                                                                 [ 36%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/configuration Failed to resolve string [{node_id}] with variables [{'protocol': 'https', 'host': 'localhost', 'port': 55000, 'user': 'testing', 'user_auth_context': 'wazuh-wui', 'pass': 'wazuh', 'basic_auth': 'dGVzdGluZzp3YXp1aA==', 'basic_auth_context': 'd2F6dWgtd3VpOndhenVoLXd1aQ==', 'login_endpoint': 'security/user/authenticate', 'reconnect_delay': 30, 'restart_delay_cluster': 150, 'upgrade_delay': 60, 'global_db_delay': 30, 'cluster_sync_delay': 20, 'custom_wpk_path': '/var/ossec/test.wpk', 'new_rules': '<!-- Local rules -->\n <group name="local,">\n    <!--   NEW RULE    -->\n    <rule id="111111" level="5">\n      <if_sid>5716</if_sid>\n      <srcip>1.1.1.1</srcip>\n      <description>sshd: authentication failed from IP 1.1.1.1.</description>\n      <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n    </rule>\n  </group>\n', 'new_rules_with_syntax_error': '<!-- Local rules -->\n <group name="local,">\n <!--   NEW RULE    -->\n <rule id="111111" level="5">\n <if_sid>5716\n <srcip>1.1.1.1</srcip>\n  <description>sshd: authentication failed from IP 1.1.1.1.</description>\n <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n </rule>\n </group>\n', 'new_decoder': '<!-- NEW Local Decoders -->\n <decoder name="local_decoder_example">\n <program_name>NEW DECODER</program_name>\n </decoder>\n', 'new_decoder_with_syntax_error': '<!-- NEW Local Decoders -->\n <decoder name="local_decoder_example" NEW DECODER</program_name>\n </decoder>\n', 'new_cdb_list': 'test-wazuh-w:write\ntest-wazuh-r:read\ntest-wazuh-a:attribute\ntest-wazuh-x:execute\ntest-wazuh-c:command\n', 'new_cdb_list_with_syntax_error': ':write\ntest-wazuh-r:read\ntest-wazuh-a:attribute\ntest-wazuh-x:execute\ntest-wazuh-c:command\n', 'corrupted_rules_file': '<!-- Local rules -->\n <group name="local,">\n <!--   NEW RULE    -->\n  <rule id="111111" level="XXX">\n      <if_sid>5716</if_sid>\n      <srcip>1.1.1.1</srcip>\n      <description>sshd: authentication failed from IP 1.1.1.1.</description>\n      <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n    </rule>\n  </group>\n', 'invalid_file': "import os #:l\nos.system('do something bad') #:l", 'file_xml': '<agent_config> <labels> <label key="aws.instance-id">i-052a1838c</label> <label key="aws.sec-group">sg-1103</label> <label key="network.ip">172.17.0.0</label> <label key="network.mac">02:42:ac:11:00:02</label> <label key="installation" hidden="yes">January 1st, 2017</label> </labels> </agent_config>', 'invalid_file_xml': 'agent_config> <labels> <label key="aws.instance-id">i-052a1838c</label> <label key="aws.sec-group">sg-1103</label> <label key="network.ip">172.17.0.0</label> <label key="network.mac">02:42:ac:11:00:02</label> <label key="installation" hidden="yes">January 1st, 2017</label> </labels> </agent_config>', 'wrong_file_xml': '<agent_confi>\n<wodle name="cis-cat"> <disabled>no</disabled> <timeout>3600</timeout> <interval>90d</interval> <scan-on-start>yes</scan-on-start>\n<java_path>/opt/java/bin</java_path> <ciscat_path>/opt/scanner</ciscat_path>\n<content type="xccdf" path="/opt/xccdf.xml"> <profile>level2</profile> </content> </wodle>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-java-check</alias> <command>fi</command> <frequency>86400</frequency> </localfile>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-scanner-check</alias> <command>fi</command> <frequency>86400</frequency> </localfile>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-ossec-log</alias> <command>grep \'wazuh-modulesd:ciscat:\' /opt/ossec/logs/ossec.log | tail -20</command> <frequency>86400</frequency> </localfile>\n</agent_confi>\n<agent_config>\n<!-- Realtime scan -->\n<syscheck>\n<!-- Full directories -->\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/logging</directories>\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/c2online</directories>\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/c2online</directories> <directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/addons</directories> <directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/commons</directories>\n\n<!-- Specific files --> <directories check_all="yes" report_changes="no" realtime="yes" restrict=".war$">/opt/deploy/c2online*.war</directories>\n<!-- Ignore folders --> <ignore type="sregex">.log$</ignore>\n</syscheck>\n</agent_config>', 'query_list_xml': "<agent_config> <localfile> <location>System</location> <log_format>eventchannel</log_format> <query> \\<QueryList> \\<Query Id='0' Path='System'> \\<Select Path='System'>*[System[(Level&lt;=3)]]\\</Select> \\</Query> \\</QueryList> </query> </localfile> </agent_config>", 'valid_ossec_conf': '<ossec_config> <global> <jsonout_output>yes</jsonout_output> <alerts_log>yes</alerts_log> <logall>no</logall> <logall_json>no</logall_json> <email_notification>no</email_notification> <agents_disconnection_time>20s</agents_disconnection_time> <agents_disconnection_alert_time>100s</agents_disconnection_alert_time> </global>\n<logging> <log_format>plain</log_format> </logging>\n<alerts> <log_alert_level>300</log_alert_level> <email_alert_level>12</email_alert_level> </alerts>\n<remote> <connection>secure</connection> <port>1514</port> <protocol>tcp</protocol> </remote>\n<syscheck> <disabled>yes</disabled> </syscheck> </ossec_config>', 'invalid_ossec_conf': '<ossec_config> <global> <jsonout_output>yes</jsonout_output> <alerts_log>yes</alerts_log> <logall>no</logall> <logall_json>no</logall_json> </global>\n<alerts> <log_alert_level>3</log_alert_level> <email_alert_level>12</email_alert_level> </alerts>\n<remote> </ossec_config>', 'test_login_token': 'eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjk3NjU1NDIwLCJleHAiOjE2OTc2NTYzMjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbOTldLCJyYmFjX21vZGUiOiJibGFjayJ9.AGcn5EMLFx7BfZGWFv20Pnj63mjS01MQKzezkhh04cVbk_u441UtAC8wOKYD5qw9yrXvUWb-cGpkYxGMc0LISThEANY2nG8_YmuNPBf5aMsHxUjWZ9Jf8GZkeztL40NfedjpQPvmiGGjjFw4L2tvIWTxCkPZn1MmE1ypbU4VWD0-p2XI', 'tavern': {'env_vars': {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/wazuhFW:@/tmp/.ICE-unix/2234,unix/wazuhFW:/tmp/.ICE-unix/2234', 'QT_ACCESSIBILITY': '1', 'COLORTERM': 'truecolor', 'XDG_CONFIG_DIRS': '/etc/xdg/xdg-ubuntu:/etc/xdg', 'XDG_MENU_PREFIX': 'gnome-', 'GNOME_DESKTOP_SESSION_ID': 'this-is-deprecated', 'TERMINATOR_DBUS_PATH': '/net/tenshu/Terminator2', 'MANDATORY_PATH': '/usr/share/gconf/ubuntu.mandatory.path', 'LC_ADDRESS': 'es_AR.UTF-8', 'GNOME_SHELL_SESSION_MODE': 'ubuntu', 'LC_NAME': 'es_AR.UTF-8', 'SSH_AUTH_SOCK': '/run/user/1000/keyring/ssh', 'TERMINATOR_UUID': 'urn:uuid:f4c7158b-90d9-49bd-88b7-97684fdff68c', 'XMODIFIERS': '@im=ibus', 'DESKTOP_SESSION': 'ubuntu', 'LC_MONETARY': 'es_AR.UTF-8', 'SSH_AGENT_PID': '2196', 'GPG_TTY': '/dev/pts/0', 'GTK_MODULES': 'gail:atk-bridge', 'PWD': '/home/fdalmau/git/wazuh/api/test/integration', 'LOGNAME': 'fdalmau', 'XDG_SESSION_DESKTOP': 'ubuntu', 'XDG_SESSION_TYPE': 'x11', 'GPG_AGENT_INFO': '/run/user/1000/gnupg/S.gpg-agent:0:1', 'XAUTHORITY': '/run/user/1000/gdm/Xauthority', 'WINDOWPATH': '2', 'HOME': '/home/fdalmau', 'USERNAME': 'fdalmau', 'IM_CONFIG_PHASE': '1', 'LC_PAPER': 'es_AR.UTF-8', 'LANG': 'en_US.UTF-8', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'XDG_CURRENT_DESKTOP': 'ubuntu:GNOME', 'VIRTUAL_ENV': '/home/fdalmau/venv/integrationtest-env-3.9', 'VTE_VERSION': '6003', 'INVOCATION_ID': '62ab06cfec4843ea9eb0e40d3ad750cd', 'TERMINATOR_DBUS_NAME': 'net.tenshu.Terminator21a9d5db22c73a993ff0b42f64b396873', 'MANAGERPID': '2015', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'XDG_SESSION_CLASS': 'user', 'PYTHONPATH': ':/home/fdalmau/git/wazuh/api:/home/fdalmau/git/wazuh/framework', 'TERM': 'xterm-256color', 'LC_IDENTIFICATION': 'es_AR.UTF-8', 'DEFAULTS_PATH': '/usr/share/gconf/ubuntu.default.path', 'LESSOPEN': '| /usr/bin/lesspipe %s', 'USER': 'fdalmau', 'DISPLAY': ':0', 'SHLVL': '1', 'LC_TELEPHONE': 'es_AR.UTF-8', 'QT_IM_MODULE': 'ibus', 'LC_MEASUREMENT': 'es_AR.UTF-8', 'XDG_RUNTIME_DIR': '/run/user/1000', 'PS1': '(integrationtest-env-3.9) \\[\\e]0;\\u@\\h: \\w\\a\\]${debian_chroot:+($debian_chroot)}\\[\\033[01;32m\\]\\u@\\h\\[\\033[00m\\]:\\[\\033[01;34m\\]\\w\\[\\033[01;33m\\]$(parse_git_branch)\\[\\033[00m\\]\\$ ', 'LC_TIME': 'es_AR.UTF-8', 'JOURNAL_STREAM': '8:51432', 'XDG_DATA_DIRS': '/usr/share/ubuntu:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PATH': '/home/fdalmau/venv/integrationtest-env-3.9/bin:/home/fdalmau/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'GDMSESSION': 'ubuntu', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1000/bus', 'GIO_LAUNCHED_DESKTOP_FILE_PID': '8252', 'LC_NUMERIC': 'es_AR.UTF-8', 'OLDPWD': '/home/fdalmau/git/wazuh', '_': '/home/fdalmau/venv/integrationtest-env-3.9/bin/pytest', 'PYTEST_CURRENT_TEST': 'test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/configuration (call)'}, 'request_vars': {'method': 'GET', 'url': 'https://localhost:55000/cluster/worker1/configuration', 'headers': {'Authorization': 'Bearer eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjk3NjU1NDIwLCJleHAiOjE2OTc2NTYzMjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbOTldLCJyYmFjX21vZGUiOiJibGFjayJ9.AGcn5EMLFx7BfZGWFv20Pnj63mjS01MQKzezkhh04cVbk_u441UtAC8wOKYD5qw9yrXvUWb-cGpkYxGMc0LISThEANY2nG8_YmuNPBf5aMsHxUjWZ9Jf8GZkeztL40NfedjpQPvmiGGjjFw4L2tvIWTxCkPZn1MmE1ypbU4VWD0-p2XI'}, 'verify': False, 'stream': False, 'allow_redirects': False}}}]
Key(s) not found in format: node_id
FAILED                                                                [ 42%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/configuration/validation PASSED                                                               [ 47%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/configuration/{component}/{configuration} PASSED                                    [ 52%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/info PASSED                                                                         [ 57%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/logs PASSED                                                                         [ 63%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/logs/summary PASSED                                                                 [ 68%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/daemons/stats PASSED                                                                [ 73%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/stats PASSED                                                                        [ 78%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/status PASSED                                                                       [ 84%]
test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/api/config PASSED                                                                             [ 89%]
test_rbac_black_cluster_endpoints.tavern.yaml::PUT /cluster/{node_id}/configuration PASSED                                                                [ 94%]
test_rbac_black_cluster_endpoints.tavern.yaml::PUT /cluster/restart PASSED                                                                                [100%]

=========================================================================== FAILURES ============================================================================
_______________ /home/fdalmau/git/wazuh/api/test/integration/test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/configuration ________________
Format variables:
Failed to resolve string [{node_id}] with variables [{'protocol': 'https', 'host': 'localhost', 'port': 55000, 'user': 'testing', 'user_auth_context': 'wazuh-wui', 'pass': 'wazuh', 'basic_auth': 'dGVzdGluZzp3YXp1aA==', 'basic_auth_context': 'd2F6dWgtd3VpOndhenVoLXd1aQ==', 'login_endpoint': 'security/user/authenticate', 'reconnect_delay': 30, 'restart_delay_cluster': 150, 'upgrade_delay': 60, 'global_db_delay': 30, 'cluster_sync_delay': 20, 'custom_wpk_path': '/var/ossec/test.wpk', 'new_rules': '<!-- Local rules -->\n <group name="local,">\n    <!--   NEW RULE    -->\n    <rule id="111111" level="5">\n      <if_sid>5716</if_sid>\n      <srcip>1.1.1.1</srcip>\n      <description>sshd: authentication failed from IP 1.1.1.1.</description>\n      <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n    </rule>\n  </group>\n', 'new_rules_with_syntax_error': '<!-- Local rules -->\n <group name="local,">\n <!--   NEW RULE    -->\n <rule id="111111" level="5">\n <if_sid>5716\n <srcip>1.1.1.1</srcip>\n  <description>sshd: authentication failed from IP 1.1.1.1.</description>\n <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n </rule>\n </group>\n', 'new_decoder': '<!-- NEW Local Decoders -->\n <decoder name="local_decoder_example">\n <program_name>NEW DECODER</program_name>\n </decoder>\n', 'new_decoder_with_syntax_error': '<!-- NEW Local Decoders -->\n <decoder name="local_decoder_example" NEW DECODER</program_name>\n </decoder>\n', 'new_cdb_list': 'test-wazuh-w:write\ntest-wazuh-r:read\ntest-wazuh-a:attribute\ntest-wazuh-x:execute\ntest-wazuh-c:command\n', 'new_cdb_list_with_syntax_error': ':write\ntest-wazuh-r:read\ntest-wazuh-a:attribute\ntest-wazuh-x:execute\ntest-wazuh-c:command\n', 'corrupted_rules_file': '<!-- Local rules -->\n <group name="local,">\n <!--   NEW RULE    -->\n  <rule id="111111" level="XXX">\n      <if_sid>5716</if_sid>\n      <srcip>1.1.1.1</srcip>\n      <description>sshd: authentication failed from IP 1.1.1.1.</description>\n      <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>\n    </rule>\n  </group>\n', 'invalid_file': "import os #:l\nos.system('do something bad') #:l", 'file_xml': '<agent_config> <labels> <label key="aws.instance-id">i-052a1838c</label> <label key="aws.sec-group">sg-1103</label> <label key="network.ip">172.17.0.0</label> <label key="network.mac">02:42:ac:11:00:02</label> <label key="installation" hidden="yes">January 1st, 2017</label> </labels> </agent_config>', 'invalid_file_xml': 'agent_config> <labels> <label key="aws.instance-id">i-052a1838c</label> <label key="aws.sec-group">sg-1103</label> <label key="network.ip">172.17.0.0</label> <label key="network.mac">02:42:ac:11:00:02</label> <label key="installation" hidden="yes">January 1st, 2017</label> </labels> </agent_config>', 'wrong_file_xml': '<agent_confi>\n<wodle name="cis-cat"> <disabled>no</disabled> <timeout>3600</timeout> <interval>90d</interval> <scan-on-start>yes</scan-on-start>\n<java_path>/opt/java/bin</java_path> <ciscat_path>/opt/scanner</ciscat_path>\n<content type="xccdf" path="/opt/xccdf.xml"> <profile>level2</profile> </content> </wodle>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-java-check</alias> <command>fi</command> <frequency>86400</frequency> </localfile>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-scanner-check</alias> <command>fi</command> <frequency>86400</frequency> </localfile>\n<localfile> <log_format>full_command</log_format> <alias>ciscat-ossec-log</alias> <command>grep \'wazuh-modulesd:ciscat:\' /opt/ossec/logs/ossec.log | tail -20</command> <frequency>86400</frequency> </localfile>\n</agent_confi>\n<agent_config>\n<!-- Realtime scan -->\n<syscheck>\n<!-- Full directories -->\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/logging</directories>\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/c2online</directories>\n<directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/c2online</directories> <directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/addons</directories> <directories check_all="yes" report_changes="yes" realtime="yes">/opt/spring/commons</directories>\n\n<!-- Specific files --> <directories check_all="yes" report_changes="no" realtime="yes" restrict=".war$">/opt/deploy/c2online*.war</directories>\n<!-- Ignore folders --> <ignore type="sregex">.log$</ignore>\n</syscheck>\n</agent_config>', 'query_list_xml': "<agent_config> <localfile> <location>System</location> <log_format>eventchannel</log_format> <query> \\<QueryList> \\<Query Id='0' Path='System'> \\<Select Path='System'>*[System[(Level&lt;=3)]]\\</Select> \\</Query> \\</QueryList> </query> </localfile> </agent_config>", 'valid_ossec_conf': '<ossec_config> <global> <jsonout_output>yes</jsonout_output> <alerts_log>yes</alerts_log> <logall>no</logall> <logall_json>no</logall_json> <email_notification>no</email_notification> <agents_disconnection_time>20s</agents_disconnection_time> <agents_disconnection_alert_time>100s</agents_disconnection_alert_time> </global>\n<logging> <log_format>plain</log_format> </logging>\n<alerts> <log_alert_level>300</log_alert_level> <email_alert_level>12</email_alert_level> </alerts>\n<remote> <connection>secure</connection> <port>1514</port> <protocol>tcp</protocol> </remote>\n<syscheck> <disabled>yes</disabled> </syscheck> </ossec_config>', 'invalid_ossec_conf': '<ossec_config> <global> <jsonout_output>yes</jsonout_output> <alerts_log>yes</alerts_log> <logall>no</logall> <logall_json>no</logall_json> </global>\n<alerts> <log_alert_level>3</log_alert_level> <email_alert_level>12</email_alert_level> </alerts>\n<remote> </ossec_config>', 'test_login_token': 'eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjk3NjU1NDIwLCJleHAiOjE2OTc2NTYzMjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbOTldLCJyYmFjX21vZGUiOiJibGFjayJ9.AGcn5EMLFx7BfZGWFv20Pnj63mjS01MQKzezkhh04cVbk_u441UtAC8wOKYD5qw9yrXvUWb-cGpkYxGMc0LISThEANY2nG8_YmuNPBf5aMsHxUjWZ9Jf8GZkeztL40NfedjpQPvmiGGjjFw4L2tvIWTxCkPZn1MmE1ypbU4VWD0-p2XI', 'tavern': {'env_vars': {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/wazuhFW:@/tmp/.ICE-unix/2234,unix/wazuhFW:/tmp/.ICE-unix/2234', 'QT_ACCESSIBILITY': '1', 'COLORTERM': 'truecolor', 'XDG_CONFIG_DIRS': '/etc/xdg/xdg-ubuntu:/etc/xdg', 'XDG_MENU_PREFIX': 'gnome-', 'GNOME_DESKTOP_SESSION_ID': 'this-is-deprecated', 'TERMINATOR_DBUS_PATH': '/net/tenshu/Terminator2', 'MANDATORY_PATH': '/usr/share/gconf/ubuntu.mandatory.path', 'LC_ADDRESS': 'es_AR.UTF-8', 'GNOME_SHELL_SESSION_MODE': 'ubuntu', 'LC_NAME': 'es_AR.UTF-8', 'SSH_AUTH_SOCK': '/run/user/1000/keyring/ssh', 'TERMINATOR_UUID': 'urn:uuid:f4c7158b-90d9-49bd-88b7-97684fdff68c', 'XMODIFIERS': '@im=ibus', 'DESKTOP_SESSION': 'ubuntu', 'LC_MONETARY': 'es_AR.UTF-8', 'SSH_AGENT_PID': '2196', 'GPG_TTY': '/dev/pts/0', 'GTK_MODULES': 'gail:atk-bridge', 'PWD': '/home/fdalmau/git/wazuh/api/test/integration', 'LOGNAME': 'fdalmau', 'XDG_SESSION_DESKTOP': 'ubuntu', 'XDG_SESSION_TYPE': 'x11', 'GPG_AGENT_INFO': '/run/user/1000/gnupg/S.gpg-agent:0:1', 'XAUTHORITY': '/run/user/1000/gdm/Xauthority', 'WINDOWPATH': '2', 'HOME': '/home/fdalmau', 'USERNAME': 'fdalmau', 'IM_CONFIG_PHASE': '1', 'LC_PAPER': 'es_AR.UTF-8', 'LANG': 'en_US.UTF-8', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'XDG_CURRENT_DESKTOP': 'ubuntu:GNOME', 'VIRTUAL_ENV': '/home/fdalmau/venv/integrationtest-env-3.9', 'VTE_VERSION': '6003', 'INVOCATION_ID': '62ab06cfec4843ea9eb0e40d3ad750cd', 'TERMINATOR_DBUS_NAME': 'net.tenshu.Terminator21a9d5db22c73a993ff0b42f64b396873', 'MANAGERPID': '2015', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'XDG_SESSION_CLASS': 'user', 'PYTHONPATH': ':/home/fdalmau/git/wazuh/api:/home/fdalmau/git/wazuh/framework', 'TERM': 'xterm-256color', 'LC_IDENTIFICATION': 'es_AR.UTF-8', 'DEFAULTS_PATH': '/usr/share/gconf/ubuntu.default.path', 'LESSOPEN': '| /usr/bin/lesspipe %s', 'USER': 'fdalmau', 'DISPLAY': ':0', 'SHLVL': '1', 'LC_TELEPHONE': 'es_AR.UTF-8', 'QT_IM_MODULE': 'ibus', 'LC_MEASUREMENT': 'es_AR.UTF-8', 'XDG_RUNTIME_DIR': '/run/user/1000', 'PS1': '(integrationtest-env-3.9) \\[\\e]0;\\u@\\h: \\w\\a\\]${debian_chroot:+($debian_chroot)}\\[\\033[01;32m\\]\\u@\\h\\[\\033[00m\\]:\\[\\033[01;34m\\]\\w\\[\\033[01;33m\\]$(parse_git_branch)\\[\\033[00m\\]\\$ ', 'LC_TIME': 'es_AR.UTF-8', 'JOURNAL_STREAM': '8:51432', 'XDG_DATA_DIRS': '/usr/share/ubuntu:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PATH': '/home/fdalmau/venv/integrationtest-env-3.9/bin:/home/fdalmau/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'GDMSESSION': 'ubuntu', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1000/bus', 'GIO_LAUNCHED_DESKTOP_FILE_PID': '8252', 'LC_NUMERIC': 'es_AR.UTF-8', 'OLDPWD': '/home/fdalmau/git/wazuh', '_': '/home/fdalmau/venv/integrationtest-env-3.9/bin/pytest', 'PYTEST_CURRENT_TEST': 'test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/configuration (call)'}, 'request_vars': {'method': 'GET', 'url': 'https://localhost:55000/cluster/worker1/configuration', 'headers': {'Authorization': 'Bearer eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjk3NjU1NDIwLCJleHAiOjE2OTc2NTYzMjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbOTldLCJyYmFjX21vZGUiOiJibGFjayJ9.AGcn5EMLFx7BfZGWFv20Pnj63mjS01MQKzezkhh04cVbk_u441UtAC8wOKYD5qw9yrXvUWb-cGpkYxGMc0LISThEANY2nG8_YmuNPBf5aMsHxUjWZ9Jf8GZkeztL40NfedjpQPvmiGGjjFw4L2tvIWTxCkPZn1MmE1ypbU4VWD0-p2XI'}, 'verify': False, 'stream': False, 'allow_redirects': False}}}]
Key(s) not found in format: node_id
  node_id = '???'
  protocol:s = 'https'
  host:s = 'localhost'
  port:d = '55000'
  test_login_token = 'eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjk3NjU1NDIwLCJleHAiOjE2OTc2NTYzMjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbOTldLCJyYmFjX21vZGUiOiJibGFjayJ9.AGcn5EMLFx7BfZGWFv20Pnj63mjS01MQKzezkhh04cVbk_u441UtAC8wOKYD5qw9yrXvUWb-cGpkYxGMc0LISThEANY2nG8_YmuNPBf5aMsHxUjWZ9Jf8GZkeztL40NfedjpQPvmiGGjjFw4L2tvIWTxCkPZn1MmE1ypbU4VWD0-p2XI'

Source test stage (line 318):
  - name: Request {node_id}
    request:
      verify: False
      url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/configuration"
      method: GET
      headers:
        Authorization: "Bearer {test_login_token}"
    response:
      status_code: 400
      json:
        error: !anyint
        data:
          affected_items:
            - alerts: !anything
              auth: !anything
              cis-cat: !anything
              cluster: !anything
              command: !anything
              global: !anything
              localfile: !anything
              osquery: !anything
              remote: !anything
              rootcheck: !anything
              ruleset: !anything
              sca: !anything
              syscheck: !anything
              syscollector: !anything
              vulnerability-detector: !anything
          failed_items: []
          total_affected_items: 1
          total_failed_items: 0


Unable to get formatted stage

Errors:
E   tavern.util.exceptions.TestFailError: Test 'Request {node_id}' failed:
    - Status code was 200, expected 400
----------------------------------------------------------------------- Captured log call -----------------------------------------------------------------------
ERROR    tavern.response.base:base.py:43 Status code was 200, expected 400
======================================================================= warnings summary ========================================================================
/home/fdalmau/venv/integrationtest-env-3.9/lib/python3.9/site-packages/tavern/testutils/pytesthook/hooks.py:43
  /home/fdalmau/venv/integrationtest-env-3.9/lib/python3.9/site-packages/tavern/testutils/pytesthook/hooks.py:43: PytestDeprecationWarning: direct construction of YamlFile has been deprecated, please use YamlFile.from_parent
    return YamlFile(path, parent)

/home/fdalmau/venv/integrationtest-env-3.9/lib/python3.9/site-packages/tavern/testutils/pytesthook/file.py:233: 19 tests with warnings
  /home/fdalmau/venv/integrationtest-env-3.9/lib/python3.9/site-packages/tavern/testutils/pytesthook/file.py:233: PytestDeprecationWarning: direct construction of YamlItem has been deprecated, please use YamlItem.from_parent
    item = YamlItem(test_spec["test_name"], self, test_spec, self.fspath)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
==================================================================== short test summary info ====================================================================
FAILED test_rbac_black_cluster_endpoints.tavern.yaml::GET /cluster/{node_id}/configuration
===================================================== 1 failed, 18 passed, 20 warnings in 131.60s (0:02:11) =====================================================
```

</details>

The usage of curly brackets in test stage names is being modified to avoid these errors.

--- a/api/test/integration/conftest.py
+++ b/api/test/integration/conftest.py
@@ -539,7 +539,7 @@ def pytest_html_results_summary(prefix, summary, postfix):
         ) for k, v in results.items()])])
 
 
-@pytest.fixture
+@pytest.fixture(scope='function', autouse=True)
 def big_events_payload() -> list:
     """Return a payload with a number of events larger than the maximum allowed.
 
@@ -551,7 +551,7 @@ def big_events_payload() -> list:
     return [f"Event {i}" for i in range(101)]
 
 
-@pytest.fixture
+@pytest.fixture(scope='function', autouse=True)
 def max_size_event() -> str:
     """Return an event with the max size allowed.
 
@@ -563,7 +563,7 @@ def max_size_event() -> str:
     return " ".join(str(i) for i in range(12772))
 
 
-@pytest.fixture
+@pytest.fixture(scope='function', autouse=True)
 def large_event() -> str:
     """Return an event with the size larger than the maximum allowed.
 

--- a/api/test/integration/test_rbac_black_cluster_endpoints.tavern.yaml
+++ b/api/test/integration/test_rbac_black_cluster_endpoints.tavern.yaml
@@ -235,7 +235,7 @@ test_name: GET /cluster/nodes/{node_id}
 
 stages:
 
-  - name: Get cluster {worker2} (Deny)
+  - name: Get cluster (worker2) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/nodes"
@@ -247,7 +247,7 @@ stages:
     response:
       <<: *permission_denied
 
-  - name: Get cluster {worker2} (Allow)
+  - name: Get cluster (worker1) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/nodes"
@@ -268,7 +268,7 @@ stages:
           total_affected_items: 1
           total_failed_items: 0
 
-  - name: Get cluster {master-node,worker1,worker2} (Allow, Deny, Allow)
+  - name: Get cluster (master-node,worker1,worker2) (Allow, Deny, Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/nodes"
@@ -316,7 +316,7 @@ test_name: GET /cluster/{node_id}/configuration
 
 stages:
 
-  - name: Request {node_id}
+  - name: Request (worker1)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/configuration"
@@ -348,7 +348,7 @@ stages:
           total_affected_items: 1
           total_failed_items: 0
 
-  - name: Request (worker1) (Deny)
+  - name: Request (master-node) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/master-node/configuration"
@@ -439,7 +439,7 @@ test_name: GET /cluster/{node_id}/info
 
 stages:
 
-  - name: Request {node_id}
+  - name: Request (worker2)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/info"
@@ -449,7 +449,7 @@ stages:
     response:
       <<: *permission_denied
 
-  - name: Request {node_id}
+  - name: Request (worker1)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/info"
@@ -464,7 +464,7 @@ test_name: GET /cluster/{node_id}/logs
 
 stages:
 
-  - name: Request {worker1} (Allow)
+  - name: Request (worker1) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/logs"
@@ -474,7 +474,7 @@ stages:
     response:
       status_code: 200
 
-  - name: Request {worker2} (Deny)
+  - name: Request (worker2) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/logs"
@@ -489,7 +489,7 @@ test_name: GET /cluster/{node_id}/logs/summary
 
 stages:
 
-  - name: Request {worker1} (Allow)
+  - name: Request (worker1) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/logs/summary"
@@ -499,7 +499,7 @@ stages:
     response:
       status_code: 200
 
-  - name: Request {worker2} (Deny)
+  - name: Request (worker2) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/logs/summary"
@@ -539,7 +539,7 @@ test_name: GET /cluster/{node_id}/stats
 
 stages:
 
-  - name: Cluster stats {worker2} today (Deny)
+  - name: Cluster stats (worker2) today (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/stats"
@@ -549,7 +549,7 @@ stages:
     response:
       <<: *permission_denied
 
-  - name: Cluster stats {worker1} today (Allow)
+  - name: Cluster stats (worker1) today (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/stats"
@@ -566,7 +566,7 @@ test_name: GET /cluster/{node_id}/status
 
 stages:
 
-  - name: Request {worker1} (Allow)
+  - name: Request (worker1) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/status"
@@ -576,7 +576,7 @@ stages:
     response:
       status_code: 200
 
-  - name: Request {worker2} (Deny)
+  - name: Request (worker2) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/status"

--- a/api/test/integration/test_rbac_white_cluster_endpoints.tavern.yaml
+++ b/api/test/integration/test_rbac_white_cluster_endpoints.tavern.yaml
@@ -233,7 +233,7 @@ test_name: GET /cluster/nodes/{node_id}
 
 stages:
 
-  - name: Get cluster {worker1} (Deny)
+  - name: Get cluster (worker1) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/nodes"
@@ -247,7 +247,7 @@ stages:
       json:
         error: 4000
 
-  - name: Get cluster {worker2} (Allow)
+  - name: Get cluster (worker2) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/nodes"
@@ -268,7 +268,7 @@ stages:
           total_affected_items: 1
           total_failed_items: 0
         
-  - name: Get cluster {master-node,worker1,worker2} (Allow, Deny, Allow)
+  - name: Get cluster (master-node,worker1,worker2) (Allow, Deny, Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/nodes"
@@ -320,7 +320,7 @@ test_name: GET /cluster/{node_id}/configuration
 
 stages:
 
-  - name: Request {node_id}
+  - name: Request (master-node)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/master-node/configuration"
@@ -465,7 +465,7 @@ test_name: GET /cluster/{node_id}/info
 
 stages:
 
-  - name: Request {node_id}
+  - name: Request (worker1)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/info"
@@ -475,7 +475,7 @@ stages:
     response:
       <<: *permission_denied
 
-  - name: Request {node_id}
+  - name: Request (master-node)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/master-node/info"
@@ -490,7 +490,7 @@ test_name: GET /cluster/{node_id}/logs
 
 stages:
 
-  - name: Request {worker2} (Allow)
+  - name: Request (worker2) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/logs"
@@ -500,7 +500,7 @@ stages:
     response:
       status_code: 200
 
-  - name: Request {worker1} (Deny)
+  - name: Request (worker1) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/logs"
@@ -515,7 +515,7 @@ test_name: GET /cluster/{node_id}/logs/summary
 
 stages:
 
-  - name: Request {worker2} (Allow)
+  - name: Request (worker2) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/logs/summary"
@@ -525,7 +525,7 @@ stages:
     response:
       status_code: 200
 
-  - name: Request {worker1} (Deny)
+  - name: Request (worker1) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/logs/summary"
@@ -565,7 +565,7 @@ test_name: GET /cluster/{node_id}/stats
 
 stages:
 
-  - name: Cluster stats {worker1} today (Deny)
+  - name: Cluster stats (worker1) today (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/stats"
@@ -575,7 +575,7 @@ stages:
     response:
       <<: *permission_denied
 
-  - name: Cluster stats {worker2} today (Allow)
+  - name: Cluster stats (worker2) today (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/stats"
@@ -592,7 +592,7 @@ test_name: GET /cluster/{node_id}/status
 
 stages:
 
-  - name: Request {worker2} (Allow)
+  - name: Request (worker2) (Allow)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker2/status"
@@ -602,7 +602,7 @@ stages:
     response:
       status_code: 200
 
-  - name: Request {worker1} (Deny)
+  - name: Request (worker1) (Deny)
     request:
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/cluster/worker1/status"

--- a/api/test/integration/test_webhook_endpoints.tavern.yaml
+++ b/api/test/integration/test_webhook_endpoints.tavern.yaml
@@ -80,12 +80,8 @@ stages:
         title: Bad Request
 
 ---
-
 test_name: POST /events with more than 100 events
 
-marks:
-  - usefixtures:
-      - big_events_payload
 
 stages:
   - name: Try to send events with an invalid size
@@ -101,13 +97,7 @@ stages:
         title: Events bulk size exceeded
 
 ---
-
 test_name: POST /events with big events
-
-marks:
-  - usefixtures:
-      - max_size_event
-      - large_event
 
 stages:
   - name: Try to send a too big event (65536B)
@@ -159,7 +149,6 @@ stages:
         error: 2
 
 ---
-
 test_name: Try to send more than 30 requests per minute
 
 stages:

